Dockerfile.template

```diff
@@ -101,7 +101,15 @@ RUN if [ -n "${TERRAFORM_VERSION}" ] && [ "${TERRAFORM_VERSION}" != "latest" ];
 RUN if [ -n "${AZ_CLI_VERSION}" ] && [ "${AZ_CLI_VERSION}" != "latest" ]; then AZ_CLI="azure-cli==${AZ_CLI_VERSION}";fi && \
```

Comment:
`azure-cli==${AZ_CLI_VERSION` this probably won't work with yum. Syntax is probably azure-cli-x.y.z